upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -381,3 +496,47 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	return addons, manifests, nil
 }
+
+func BuildSecret() (kube_api.Secret, error) {
+	secret, err := fi.CreateSecret()
+	if err != nil {
+		return kube_api.Secret{}, fmt.Errorf("error create secret: %s", err)
+	}
+	secData := make(map[string][]byte)
+	secData["weave-pass"] = []byte(secret.Data)
+	seConfig := kube_api.Secret{
+		Data: secData,
+		Type: kube_api.SecretTypeOpaque,
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "weave-pass",
+			Namespace: "kube-system"}}
+	return seConfig, err
+}
+
+func BuildNewEnv() []kube_api.EnvVar {
```

Comment:
Again do we need to make this private?  If not can we get code comments for the func?